komga-webui/src/locales/en.json

```diff
@@ -329,13 +329,13 @@
       "dialog_title_multiple": "Edit {count} book | Edit {count} books",
       "dialog_title_single": "Edit {book}",
       "field_isbn": "ISBN",
-      "field_isbn_error": "Must be a valid ISBN 13",
+      "field_isbn_error": "Must be a valid 13-digit ISBN",
```

Comment:
Makes it a little easier to understand what it is, and "ISBN 13" vs. "ISBN-13" is avoided.